test/cli_create_test.go

```diff
@@ -608,3 +608,12 @@ func (suite *PouchCreateSuite) TestCreateWithoutNvidiaConfig(c *check.C) {
 	}
 	c.Assert(result[0].HostConfig.Resources.NvidiaConfig, check.IsNil)
 }
+
+// TestCreateWithInvalidName tests creating container with invalid name.
+func (suite *PouchRunSuite) TestCreateWithInvalidName(c *check.C) {
+	name := "new:invalid"
+	res := command.PouchRun("create", "--name", name, busyboxImage)
+	if !strings.Contains(res.Stdout(), "Invalid container name") {
```

Comment:
clean the container `defer DelContainerForceMultyTime(c, name)`

Reply:
OK!